CONTRIBUTING.md

```diff
@@ -120,6 +122,13 @@ If you are changing only a macro, the commit will look like this:
 rule(macro user_known_write_monitored_dir_conditions): make sure conditions are great
 ```
 
+## Coding Guidelines
```

Comment:
Ooooh, thank you very much @vrongmeal! 🤗 

This is exactly what I meant :)

Reply:
Updated the organisation-wide CONTRIBUTING.md file too -> https://github.com/falcosecurity/.github/pull/8

Thanks!